test/mock-server.js

```diff
@@ -126,7 +126,7 @@ api.createServer = () => {
       keyDoc.publicKeyPem = publicKeyPem;
     }
     if(revoked) {
-      keyDoc.revoked = true;
+      keyDoc.revoked = new Date();
```

Comment:
This will fail the just added check to ensure revoked is a string.  Probably should add some tests using revoked (or note that they should be added).

Reply:
Test is passing: https://github.com/digitalbazaar/http-signature-middleware/blob/implementation/test/test.js#L173